securemailbox/views/register.py

```diff
@@ -9,4 +12,43 @@
 
 @register_blueprint.route("/register/", methods=["POST"])
 def register():
-    return jsonify({"success": True, "error": None})
+    # Ensure a valid request is received
+    if not request.is_json:
+        return jsonify({"success": False, "error": "Request must be valid json"}), 400
+
+    # Ensure all fields in request are not NULL
+    for field in ["fingerprint"]:
+        field_to_check = request.json.get(field, None)
+        if field_to_check is None:
+            return (
+                jsonify({"success": False, "error": f"field '{field}' is required."}),
+                400,
+            )
+        if len(field_to_check) > FINGERPRINT_LENGTH:
```

Comment:
I think once we determine the length that fingerprints will be we should reject anything that isn't exactly that length but for now this seems fine.